dynamic-html.js

```diff
@@ -0,0 +1,164 @@
+// section Created
+
+const arr = [
+  {
+    close: './project-images/Icon.png',
+    logo: './project-images/one.png',
+    title: 'Random name for project 1',
+    tech: ['Ruby', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum recusandae accusamus minima et officiis vel, unde cum saepe officia, iusto eaque doloremque commodi quisquam rem nisi error illum ipsa repellat!',
```

Comment:
- [ ] On the modal pop-up kindly add padding to the project description.

<img src="https://user-images.githubusercontent.com/33323077/131946457-180e85ad-5ff7-47fa-bde3-fb9560423d6e.png" height="200px"/>

Reply:
I fixed that issue thank you very much